community/codegen/src/main/java/org/neo4j/codegen/bytecode/Block.java

```diff
@@ -17,22 +17,11 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.codegen;
+package org.neo4j.codegen.bytecode;
 
-public class TryBlock extends CodeBlock
-{
-    TryBlock( CodeBlock parent )
-    {
-        super( parent );
-    }
-
-    public CodeBlock catchBlock(Parameter exception)
-    {
-        return emitCatch(exception);
-    }
+import org.objectweb.asm.Opcodes;
 
-    public CodeBlock finallyBlock()
-    {
-        return emitFinally();
-    }
+interface Block extends Opcodes
```

Comment:
I'm not a fan of bringing in constants by implementing an interface. `import static  org.objectweb.asm.Opcodes.*` would be preferable IMO.